package.json

```diff
@@ -1,19 +1,24 @@
 {
   "name": "http-traceroute",
   "version": "1.6.1",
+  "main": "lib/trace.js",
   "description": "A command line tool for following and showing HTTP redirects for a given URL",
-  "bin": "index.js",
+  "bin": {
```

Comment:
Yeah, I know, I had some issues with the automatic `bin` mapping on Windows some npm versions ago, and since then I like to be explicit with it :) But I can change it, if you like?